### PR TITLE
feat(colors, typography): Added stories for colors and typography

### PR DESCRIPTION
**Pull request by cozzbp**
image

image

### CODE REVIEW

File: src/components/styles/typography.stories.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+storiesOf('Base', module)
+.addWithChapters(
+  'Typography',
+  {
+    info: `
+      Usage
+
+      ~~~
+      import React from 'react';
+      import { Icons } from 'insidesales-components';
```
Comment (seanevd) [inline src/components/styles/typography.stories.js]: Import typography

File: src/components/styles/typography.stories.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  flex-wrap: wrap;
+  align-items: center;
+`;
+
+const TypographyWrapper = styled.div`
+  margin: 20px;
+  ${props => props.extraStyle};
+`;
+
+const icons = _.map(typography, (type, key) => {
```
Comment (seanevd) [inline src/components/styles/typography.stories.js]: this should also be typography, I think

File: src/components/styles/colors.stories.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+storiesOf('Base', module)
+.addWithChapters(
+  'Colors',
+  {
+    info: `
+      Usage
+
+      ~~~
+      import React from 'react';
+      import { Icons } from 'insidesales-components';
```
Comment (seanevd) [inline src/components/styles/colors.stories.js]: This should be `colors`

File: src/components/styles/typography.stories.js (hunk 1 above)
Comment (cozzbp) [reply src/components/styles/typography.stories.js]: Good catch.  Fixed.

**Comment by seanevd** [comment]
Why are you waiting to merge this?

**Comment by cozzbp** [comment]
I thought I already merged it.  Updating with master now.

**Pull request merged**